src/firmware_new/boards/frankie_v1/io/infineon_TLE5009_E1000.c

```diff
@@ -0,0 +1,70 @@
+#include "infineon_TLE5009_E1000.h"
+
+#include <stdlib.h>
+
+struct InfineonTLE5009E1000
+{
+    float A_x;
+    float A_y;
+    float O_x;
+    float O_y;
+    float phase_offset_x_rad;
+    float phase_difference_offset_rad;
+};
+
+InfineonTLE5009E1000_t* io_infineon_TLE5009_E1000_create(
+    float x_max_value, float x_min_value, float y_max_value, float y_min_value,
+    float x_magnitude_45_degrees, float y_magnitude_45_degrees,
+    float x_magnitude_135_degrees, float y_magnitude_135_degrees)
+{
+    // This function intentionally uses variable names identical to the sensor data-sheet
```

Comment:
Could we link to the datasheet here? Preferably with a page number people could look at.

Reply:
Done

Reply:
Formatting seems a little funky on the comment? Generally for in-function comments we use `//` to comment each line.

Reply:
Fixed this